dev-docs/explanations/string_changes_policy.md

```diff
@@ -0,0 +1,30 @@
+# Policy on changing user-facing strings in Ubuntu Pro Client
+
+As of version 30 of `pro`, we support translations of user-facing strings throughout the program.
+These are maintained in `po` files in the `debian/po` directory of the git repository.
+Whenever a string is changed, added, or deleted, that has an impact on existing translations and
+therefore an impact on the user experience of `pro` in non-english locales.
+To minimize regressions in translations, we have the following policy in place regarding string changes.
+This policy is implemented in practice via a custom Github action that posts a questionare and prompt
+for justification on all Github PRs that modify the `messages` module of Ubuntu Pro Client.
+
+## Changing existing (already translated) strings
+
+All changes to existing strings will require justification in the Github PR that introduces the change.
+
+Acceptable justifications:
+- Typos: these will hopefully be minimal after adding spell-check CI, but still could occur.
```

Comment:
When is/was spell-check CI added? Not clear if this is already implemented or due to appear in the future

Reply:
It will be added by the time this goes live :), I'll clarify that in the text